File: CPP/fizzbuzz.cpp
#include <iostream>
#include <string>


using namespace std;


int main()
{
    int num1 = 3;
    int num2 = 5;
    int num3 = 15;
    cin >> num1 >> num2 >> num3;
    
    
    for (int i =1; i <= num3; i++)
    {
        if (i% num2 == 0 && i% num1 == 0) {
            cout << "fizzbuzz" << endl;
        }
        else if (i% num1 == 0){
            cout << "fizz" << endl;
        }
        else if (i% num2 == 0) {
            cout << "buzz" << endl;
        }
        else {
            cout << i << endl;
        }
    }
    system("pause");
    return 0;
}

File: PY/fizzbuzz.py
#!usr/bin/python3


input = input()
x, y, n = input.split()
#print(x, y, n)
x = int(x) #2
y = int(y) #3
n = int(n) #7
for i in range(1, n + 1):
    if i % x == 0 and i % y == 0: print('fizzbuzz')
    elif i % x == 0: print('Fizz')
    elif i % y == 0: print('Buzz')
    else: print(i)